prometheus_hardware_exporter/collector.py

```diff
@@ -349,11 +360,39 @@ def fetch(self) -> List[Payload]:
             logger.error("Failed to fetch current power from ipmi dcmi")
             return [Payload(name="ipmi_dcmi_command_success", value=0.0)]
 
+        get_ps_redundancy_ok, ps_redundancy = self.ipmi_tool.get_ps_redundancy()
+        # Because we fail to get the redundancy config from the server,
+        # Suppose redundancy enable make denominator smaller
+        # and alert is more easy to fire.
+        if not get_ps_redundancy_ok:
+            ps_redundancy = True
+
+        power_capacities = self.dmidecode.get_power_capacities()
+        maximum_power_capacity = (
```

Comment:
Please add some comment to explain the fomula. It's different when ps_redundancy is true/false.